packages/protocol/contracts-0.8/common/EpochManager.sol

```diff
@@ -326,6 +326,18 @@ contract EpochManager is
     );
   }
 
+  /**
+   * @notice Returns the epoch info of a specified epoch.
+   * @param epochNumber Epoch number where epoch info is retreived.
+   * @return Epoch info.
```

Comment:
Both this and `getCurrentEpoch` could use a more detailed NatSpec with each return value described.